packages/eslint-plugin/src/rules/no-unexplained-console-error/no-unexplained-console-error.ts

```diff
@@ -0,0 +1,45 @@
+import { createRule } from '../../utils'
+
+export const messages = {
+    noUnexplainedConsoleError:
+        'Directly logging through `console.error()` is discouraged. If required, please add a comment explaining why so, otherwise consider passing the error through `Sentry.captureException()` instead.',
```

Comment:
Depending on the outcome of [the discussion](https://github.com/sourcegraph/sourcegraph/pull/38475#discussion_r917562173) in the follow-up PR, we would need to update the message here. Using `Sentry.captureException` directly is not safe because it's not always available. We would want to point consumers to something more generic that encapsulates Sentry and other output channels.